packages/sanity/src/core/form/__telemetry__/form.telemetry.ts

```diff
@@ -14,7 +14,7 @@ export const PortableTextInputCollapsed = defineEvent({
 
```

Comment:
A comment about the structure of the names here could help in the future:
`“<object/noun> <action/verb>”`